v2/pink-sb/src/lib/input/Helper.svelte

```diff
@@ -49,5 +51,10 @@
         &.error {
             color: var(--color-fgcolor-error);
         }
+
+        & span :global(i) {
+            // text's line-height: 140%; /* 19.6px */
+            --p-icon-size: 19.6px !important;
+        }
```

Comment:
we should try and avoid `:globals` 👍🏻

Reply:
you can pass them like this to the component 👍🏻 https://svelte.dev/docs/svelte/custom-properties